scene/3d/physics_body.cpp

```diff
@@ -92,17 +136,42 @@ void PhysicsBody::remove_collision_exception_with(Node* p_node) {
 	PhysicsServer::get_singleton()->body_remove_collision_exception(get_rid(),physics_body->get_rid());
 }
 
+void PhysicsBody::_set_layers(uint32_t p_mask) {
```

Comment:
Why isn't is shown in the class ref? And that's precisely because it works for ``set("layers", value)`` that a deprecation warning could be useful, so that developers using this know that they should switch to ``set_layer_mask(value)`` and ``set_collision_mask(value)`` ;)